HISTORY.md

```diff
@@ -1,3 +1,19 @@
+# v0.5.0 - Jul ??, 2020
+
+The major change in this release is the move from operator SDK v0.8.1 to v0.15.2, which has a couple of notable sets of consequences:
+* Anyone building KubeDirector from source, as opposed to using a pre-built operator image, should re-read [doc/kubedirector-development.md](doc/kubedirector-development.md) to become acquainted with new requirements and with some tips for continuing to build your existing local repo. We highly recommend reading this **before** merging the new release source into your existing source.
+* Anyone who is currently running a KubeDirector 0.4.x deployment and is interested in updating that deployment in-place should read [doc/upgrade.md](doc/upgrade.md).
+
+KD 0.5.0 supports some additional properties in [KubeDirectorApp](https://github.com/bluek8s/kubedirector/wiki/KubeDirectorApp-Definition) and [KubeDirectorCluster](https://github.com/bluek8s/kubedirector/wiki/KubeDirectorCluster-Definition); see the "properties supported by KD v0.5.0+" sections in those tables. These implement three new features:
+* A KD app can indicate that a unique token should be generated to be used for authentication to a service endpoint. This token will be visible in the KD cluster member status, as well as visible within the relevant container (through configcli) for use in setting up that service.
```

Comment:
the token is also available in service object's annotation

Reply:
Can you give me more detail about that? I can test later, but if you know off the top of your head that would be handy.

Reply:
Let's say in the KDApp, jupyter notebook service of type NodePort has **hasAuthToken** set to True, then we generate this annotation in the svc Object
```
k8suser@kdss-tsxvh-0:~$ kubectl get svc s-kdss-4nbf8-0 -o yaml
apiVersion: v1
kind: Service
metadata:
  annotations:
    hpecp-internal-gateway/22: mip-bd-vm140.mip.storage.hpecorp.net:10006
    hpecp-internal-gateway/8888: mip-bd-vm140.mip.storage.hpecorp.net:10007
    kubedirector.hpe.com/kd-auth-token: 12df044ef8d1774c8c8772dafe6082b9
```

Reply:
Added detail to the wiki; I'll also update this with a general comment.